pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -344,7 +344,7 @@ func NewNodeLifecycleController(
 		secondaryEvictionLimiterQPS: secondaryEvictionLimiterQPS,
 		largeClusterThreshold:       largeClusterThreshold,
 		unhealthyZoneThreshold:      unhealthyZoneThreshold,
-		nodeUpdateQueue:             workqueue.NewNamed("node_lifecycle_controller"),
+		nodeUpdateQueue:             workqueue.NewTypedWithConfig[any](workqueue.TypedQueueConfig[any]{Name: "node_lifecycle_controller"}),
```

Comment:
Should we add a TODO to convert this to use `TypedQueueConfig[Node]` ?

Reply:
Also, this can be simpler: `workqueue.NewTypedWithConfig(workqueue.TypedQueueConfig[any]{Name: "node_lifecycle_controller"})`

Reply:
I am happy to see that we started to use the typed workq, but why did we type it to `any`? That makes it basically pointless. Also xref https://github.com/kubernetes/kubernetes/pull/124600 which does this throughout the repo